reana_client/cli/ping.py

```diff
@@ -43,9 +43,13 @@ def ping(ctx, access_token):  # noqa: D301
         click.echo(
             click.style(
                 "REANA server: {0}\n"
-                "Authenticated as: {1} <{2}>\n"
-                "Status: {3}".format(
+                "REANA server version: {1}\n"
+                "REANA client version: {2}\n"
+                "Authenticated as: {3} <{4}>\n"
+                "Status: {5}".format(
                     get_api_url(),
+                    response.get("reana_server_version") or "",
```

Comment:
```suggestion
                    response.get("reana_server_version", ""),
```

Reply:
@mvidalgarcia 

I added `""` after this comment https://github.com/reanahub/reana-client/pull/442#discussion_r518647376

If this doesn't address the comment, I can remove it.

Reply:
It's not removing the `""` but making it the default value if `"reana_server_version"` doesn't exist.
My point is that you don't need the `or`, both expressions are equivalent.

Reply:
Yes, got it. Thanks for pointing it out, I corrected one more similar thing.